lib/config/options/index.ts

```diff
@@ -68,6 +68,33 @@ const options: RenovateOptions[] = [
     default: false,
     globalOnly: true,
   },
+  {
+    name: 'mergeConfidenceEndpoint',
+    description:
+      'If set, Renovate will query this API for Merge Confidence data.',
+    type: 'string',
+    default: 'https://developer.mend.io/',
+    advancedUse: true,
+    globalOnly: true,
+  },
+  {
+    name: 'mergeConfidenceDatasources',
+    description:
+      'If set, Renovate will query the merge-confidence JSON API only for datasources that are part of this list.',
```

Comment:
@nabeelsaabna can you remind me why this option is needed? e.g. in which cases would a user want to restrict the list?

Reply:
I think it's to override from hosted app, when new datasources are added and renovate don't know them yet

Reply:
i think this should use `supportedDatasources` from merge-confidence preset.